local.tf

```diff
@@ -0,0 +1,566 @@
+terraform {
+  required_version = "~> 0.12.25"
+  backend "local" {}
+}
+
+provider "aws" {
+  region = "us-west-2"
```

Comment:
Should this be in a separate variables file, how does TF support various optional settings?

Reply:
I have some variables below. I can add one for region while keeping this as the default. At apply time, variables can be specified inline in the apply command or in a .tfvariables file.